test/cli/fuzz-crash/crash-039028704ca27187fc3228e9fa01ca30a8434e7a

```diff
@@ -0,0 +1 @@
+_ 1p;
```

Comment:
this is garbage code and ideally I think we would throw a syntaxError. That might prevent further crashes if the fuzzer tries more such garbage..

I don't know .. a `+ name literal` statement might be garbage always?